l10n_ar_afipws/views/afipws_certificate_alias_view.xml

```diff
@@ -6,30 +6,30 @@
         <field name="model">afipws.certificate_alias</field>
         <field name="arch" type="xml">
             <form string="AFIP Certificates">
-                <field name="service_type" invisible="1"/>
+                <field name="service_type" column_invisible="1"/>
 
                 <header>
                     <button name="action_confirm"
                         type="object"
-                        states="draft"
+                        invisible="state != 'draft'"
                         string="Confirm"
                         class="oe_highlight"
                         help="Confirm and generate Key"
                         />
                     <button name="action_create_certificate_request"
                         type="object"
-                        states="confirmed"
+                        invisible="state != 'confirmed'"
                         class="oe_highlight"
                         string="Create Certificate Request"
                         />
                     <button name="action_cancel"
                         type="object"
-                        states="draft,confirmed"
+                        invisible="state != 'draft' and state != 'confirmed')"
```

Comment:
Para estos casos preferimos not in , quedaría algo así
`invisible="state not in ('draft', 'confirmed')"`